health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/SelectingBoundaryComponent.js

```
@@ -444,163 +444,169 @@ const SelectingBoundaryComponent = ({
+              const frozenCount = frozenData?.filter((frozenItem) => frozenItem?.type === boundary.boundaryType).length;
+
+              // Find the count of boundaryType in boundaryOptions
+              const optionsCount = Object.entries(boundaryOptions)
+                .filter(([key]) => key.startsWith(boundary.boundaryType))
+                .flatMap(([key, value]) => Object.entries(value || {})).length;
+
+              // Disable dropdown if counts match
+              const isDisabled = frozenCount === optionsCount;
+
+              return boundary?.parentBoundaryType == null ? (
+                <LabelFieldPair style={{ alignItems: "flex-start", paddingRight: "30%" }}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)